kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/S3SinkConnectorNetworkIT.java

```diff
@@ -0,0 +1,334 @@
+import static org.apache.kafka.connect.runtime.ConnectorConfig.*;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Integration test for S3 Sink Connector.<p>
+ * Refer file /bin/README.md to find prerequisites
+ * to run the following tests.
+ */
+@Category(IntegrationTest.class)
+public class S3SinkConnectorNetworkIT extends BaseConnectorNetworkIT {
+
+  private static final Logger log = LoggerFactory.getLogger(S3SinkConnectorIT.class);
+
+  private static final String CONNECTOR_NAME = "s3-sink-connector";
+  private static final String STORAGE_CLASS_CONFIG = "storage.class";
+  private static final int NUM_RECORDS_PRODUCED = 1000;
+  private static final int TASKS_MAX = 1;
+  private static final String TEST_TOPIC_NAME = "topic1";
+  private static final String EXPECTED_FILE_EXTENSION = "json";
+  private static final List<String> KAFKA_TOPICS = Arrays.asList(TEST_TOPIC_NAME);
+  private static final int FLUSH_SIZE = 200;
+  private static final int EXPECTED_PARTITION = 0;
+  private static SquidProxy squid;
+
+  @Before
+  public void initializeResources() {
+    createS3RootClient();
+    // create the test bucket
+    createS3Bucket(TEST_BUCKET_NAME);
+  }
+
+
+  @AfterClass
+  public static void deleteBucket() {
+    // delete the test bucket
+    deleteBucket(TEST_BUCKET_NAME);
+  }
+
+  /**
+   * Success scenario : End to end test
+   * @throws InterruptedException
+   */
+  @Test
+  @Ignore
+  public void testToAssertConnectorAndDestinationRecords() throws InterruptedException {
+
+    // create topics in Kafka
+    KAFKA_TOPICS.forEach(topic -> connect.kafka().createTopic(topic, 1));
+    // send records to kafka
+    sendRecordsToKafka(NUM_RECORDS_PRODUCED);
+
+    // start a sink connector
+    connect.configureConnector(CONNECTOR_NAME, getConnectorProps());
+    // wait for tasks to spin up
+    int minimumNumTasks = Math.min(KAFKA_TOPICS.size(), TASKS_MAX);
+
+    waitForConnectorToStart(CONNECTOR_NAME, minimumNumTasks);
+    int expectedFileCount = NUM_RECORDS_PRODUCED / FLUSH_SIZE;
+    waitForFilesInBucket(TEST_BUCKET_NAME, expectedFileCount);
+
+    // assert files in bucket
+    List<String> expectedFilenames = getExpectedFilenames(TEST_TOPIC_NAME, EXPECTED_PARTITION,
+        FLUSH_SIZE, NUM_RECORDS_PRODUCED, EXPECTED_FILE_EXTENSION);
+    assertTrue(fileNamesValid(TEST_BUCKET_NAME, expectedFilenames));
+  }
+
+  /**
+   * Case in which bucket permissions/policies are changed while uploading records
+   * Prerequisite : Access key and Secret access key should be set as environment variables
+   * @throws InterruptedException
+   */
+  @Test
+  @Ignore
+  public void testWithRevokedWritePermissions() throws InterruptedException {
+
+    addReadWritePolicyToBucket(TEST_BUCKET_NAME);
+    KAFKA_TOPICS.forEach(topic -> connect.kafka().createTopic(topic, 1));
+
+    // send records to kafka
+    sendRecordsToKafka(NUM_RECORDS_PRODUCED);
+
+    props = getConnectorProps();
```

Comment:
We won't need these in each of the tests after the new change of using the global.